### PR TITLE
Fix some broken internal links on some pages

### PR DESCRIPTION
**Pull request by serg**

### CODE REVIEW

File: themes/kubermatic-docs/layouts/_default/_markup/render-image.html [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
 <figure>
-  <img src="{{ .Destination | safeURL }}" title="{{ .Title }}">
+  <img src="{{ .Destination | safeURL }}" title="{{ .Title }}"{{with .Text}} alt="{{.}}"{{end}}/>
   <figcaption>
     <h4>{{ .Title }}</h4>
   </figcaption>
 </figure>
 {{ else }}
-<img src="{{ .Destination | safeURL }}" alt="{{ .Text }}">
+<img src="{{ .Destination | safeURL }}"{{with .Text}} alt="{{.}}"{{end}}/>
```
Comment (embik) [inline themes/kubermatic-docs/layouts/_default/_markup/render-image.html:9]: Minor nit, but I don't think the trailing slash does anything in the `img` tag anymore? Should we really add it?

Comment (serg) [reply themes/kubermatic-docs/layouts/_default/_markup/render-image.html]: Yeah, the slash is optional. I think this is my habit to mark self closing tags with slash at end 😄 . Now, removed from here.

**Review by embik — APPROVED** [review]
/approve

**Pull request merged**